applications/tari_collectibles/README.md

```diff
@@ -2,16 +2,15 @@
 
 ```bash
 # run a tari_console_wallet with grpc on port 18143
+# run a base node with grpc on port 18142
+# run a validator node with grpc on port 18144
```

Comment:
This needs some more info:
- The validator node section in `tari_config_example.toml` (imported from `validator_node.toml`) has information missing, for example the `[validator_node]` divider and I assume `grpc_enabled` and `grpc_address`.
- General  validator node command line options
- It seems to clash with a running base node? - getting `database locked` error